### DISCUSSION TITLE
What's the purpose of showing icons in tray tooltips?

### DISCUSSION THREAD

**Discussion by error256** [bbidulock/icewm#691] [Q&A]
IceWM 3.2.3 shows big icons in tooltips for apps in the tray. I can't think of any reason for that, but I rarely use advanced features. Can an icon in the tooltip be different from the icon in the tray in some situation of what is it there for?

**Comment by gijsbers** [comment]
See #637 for the idea. Imagine a high-resolution laptop screen with tiny icons where a tooltip with a large icon asserts what the button is about.
